infinite-tracing/src/test/java/com/newrelic/ResponseObserverTest.java

```diff
@@ -98,7 +90,7 @@ void shutdownChannelAndBackoff_ConnectTimeoutBackoffZeroSeconds() {
 
         target.shutdownChannelAndBackoff(status);
 
-        verify(logger).log(eq(Level.FINE), any(Throwable.class), anyString(), any());
+        verify(logger).log(eq(Level.FINE), any(Throwable.class), anyString(), anyString());
```

Comment:
any() was failing.  anyString() was a better arguement matcher and passed the test.